packages/main/src/Table.js

```diff
@@ -182,12 +187,8 @@ class Table extends UI5Element {
 		});
 	}
 
-	onEnterDOM() {
```

Comment:
I think we still need this because the element may be removed and put back to dom many times and once it's deregistered on onExitDOM, it will never be registered again